Updated README.md

amzamani opened the pull request:
- Fixel clone repo URL
- Fixed Licence URL
- Fixed img shields, license logo

## Description


This PR fixes the URL typos in the readme file which were caused due to name changes.


## What type of PR is this? (check all applicable)

- [ ] 💡 Feature
- [ ] 🐛 Bug Fix
- [X] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

<!--
Please use this format link issue numbers: Fixes #123
https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
-->

## Mobile & Desktop Screenshots/Recordings

The readme screenshot : 
![image](https://github.com/projectx-codehagen/Badget/assets/62957837/ca0db806-7b44-4851-8d4f-5cedcbcb98df)


<!-- Visual changes require screenshots -->

## Steps to QA

<!--
Please provide some steps for the reviewer to test your change. If you have wrote tests, you can mention that here instead.

1. Click a link
2. Do this thing
3. Validate you see the thing working
-->

1. Click the License image link to validate its working
2. Git clone repo command can be copied and cloned with correct URL

## Added to documentation?

- [X] 📜 README.md
- [ ] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

<!-- note: PRs with deleted sections will be marked invalid -->

<!--
  For Work In Progress Pull Requests, please use the Draft PR feature,
  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.

  For a timely review/response, please avoid force-pushing additional
  commits if your PR already received reviews or comments.

  Before submitting a Pull Request, please ensure you've done the following:
  - 👷‍♀️ Create small PRs. In most cases, this will be possible.
  - ✅ Provide tests for your changes.
  - 📝 Use descriptive commit messages.
  - 📗 Update any related documentation and include any relevant screenshots.
-->

meglerhagen reviewed (changes requested):
Looks good. Could you also else where it says badget -> Badget in the readme?

amzamani commented:
> Looks good. Could you also else where it says badget -> Badget in the readme?

- Converted "badget" -> "Badget"
- Updated with correct URLs in Contributing section

meglerhagen reviewed (approved):
Looks good! Thank you ✨
